src/test/java/net/mcreator/integration/ui/UITestUtil.java

```diff
@@ -45,7 +45,7 @@ public static void waitUntilWindowIsOpen(Window master, Runnable openTask) throw
 			//noinspection BusyWait
 			Thread.sleep(100);
 
-			if (System.currentTimeMillis() - start > 6000)
+			if (System.currentTimeMillis() - start > 7000)

```

Comment:
Any specific reason for this change? No UI should take 7s to open

Reply:
If tests sometimes fail due to this, it is due to bad runner assigned by GH fixed by test re-roll and not by increasing this value